lib/game_session.rb

```diff
@@ -1,12 +1,12 @@
 class GameSession
```

Comment:
FYI – you can simplify your `switch_turns`: @current_player = (@current_player = player1) ? player2 : player1;

Reply:
I simplified it to this: @current_player = (@current_player == player2) ? player1 : player2. I can't figure out why changing the code to your version doesn't switch turns. I also tried it like this: @current_player = (@current_player == player1) ? player2 : player1

Reply:
Did the last version above work? Malina accidentally left a `=` out in the conditional. 😛  A single `=` represents an assignment, whereas `==` represents a conditional for equality.

Reply:
LOL whoops.

Reply:
I caught her = typo and changed it to == but the code still didn't switch turns